backend/db/utils.py

```diff
@@ -626,18 +638,23 @@ def load_csv(
         update_db_status_var(f'last_updated_{table}', str(current_datetime()), local)
 
 
-def list_tables(con: Connection, schema: str = SCHEMA, filter_temp_refresh_tables=False) -> List[str]:
+def filter_temp_refresh_tables(tables: List[str]) -> List[str]:
```

Comment:
## `filter_temp_refresh_tables()`
I could have just had the same code in 2 places to deal w/ this but I decided to DRY it.